README.md

```diff
@@ -1,29 +1,60 @@
-# Math-Magician
+# 📖 [Bookstore] <a name="Bookstore: connect to API"></a>
 
-This is a book store app build by react library.
+> n this project, I connected my React-Redux application to an existing API to create and remove books in a remote server.
 
-![SCREENSHOT](./src/component/images/)
+**[Bookstore]** connect to API
 
-## Built With
+## 🛠 Built With <a name="built-with"></a>
 
-- ReactJS
+### Tech Stack <a name="tech-stack"></a>
 
-## Video Walkthrough
+> HTML, CSS, JS Webpack, react-redux.
 
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
 
+<!-- Features -->
 
-## Get Live Demo
+### Key Features <a name="Conneting to api server"></a>
 
-[Live Demo Netlify Link]()
+> Post and get data from api server.
 
+- **[Post]**
+- **[Get]**
+- **[Store]**
 
-## Clone and review it
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Run the following commands to clone and run it.
+<!-- LIVE DEMO -->
 
-### To clone the repository
+## 🚀 Live Demo <a name="live-demo"></a>
 
-  `git clone https://github.com/eerapheal/my-math-apps.git`
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+Vscode. must have git installed.
+In order to run this project you need:
+
+
+### Setup
+
+Clone this repository to your desired folder:
 
 ### To enter Directory
 
```

Comment:
- [ OPTIONAL ] I think it would be better if you add a command on how to clone your project locally. I can see that you started with the command of how to access the directory without showing to clone the repo. Adding that will make your documentation looks again more professional.